.codespellrc

```diff
@@ -0,0 +1,2 @@
+[codespell]
+skip = ./.git,./build,./envs,./usbmuxctl.egg-info
```

Comment:
I did not quite copy the `.gitignore` (mostly because all of these entries in the same line become somewhat unwieldy), but I've added all env/venv/envs from the `.gitignore` now.